File: MQTTCSharpExample/Structs/MQTTRunFolder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MQTTCSharpExample
{
    public class MQTTRunFolder
    {
        public string RunName;
        public string RunPath;
        public List<string> RunFiles;
    }

    public class MQTTRecordFile
    {
        public string FileName;
        public byte[] AtfxFileContent;
        public List<string> DataFiles;
        public List<byte[]> DataFileContents;
    }
}

File: MQTTJavaDemoCode/JavaSwingDemoMqtt GUI app/MQTTCSharpProj/src/main/java/org/example/messages/MqttRunFolder.java
package org.example.messages;

import java.util.List;

public class MqttRunFolder {
    public String RunName;
    public String RunPath;
    public List<String> RunFiles;
    public MqttRunFolder(){

    }

    public MqttRunFolder(String runName, String runPath, List<String> runFiles) {
        RunName = runName;
        RunPath = runPath;
        RunFiles = runFiles;
    }
}
